test/integration/addons_test.go

```diff
@@ -131,7 +131,11 @@ func TestAddons(t *testing.T) {
 
 	// AutoPause addon test
 	t.Run("AutoPause", func(t *testing.T) {
-		validateAutoPause(ctx, t, profile)
```

Comment:
!= docker

Reply:
@medyagh , already skipped the test for `docker` cri